department-of-veterans-affairs/caseflow
# Hearing Worksheet | Case Details Link

Resolves #8592

### Description
This pr adds a Case detail  link on the Hearing worksheet 

### Acceptance Criteria
- [ ] On the hearing worksheet, for both legacy and ama appeals, we need to add a button in the top righthand corner that links to the Case Details page. We should use the Link component stylized as a button.

### Testing Plan
1. Go to Hearing worksheet and see the Case details link.
<img width="1455" alt="screen shot 2019-01-10 at 12 03 23 pm" src="https://user-images.githubusercontent.com/23080951/50985558-9d85f380-14d2-11e9-8635-dea813b8678d.png">

2. Click on the `View case detail` link and see if it takes you to Case details page.
